geoips/plugins/modules/procflows/single_source.py

```diff
@@ -1660,9 +1660,9 @@ def call(fnames, command_line_args=None):
         )
 
     print_mem_usage("MEMUSG", verbose=True)
-    LOG.info("READER_NAME: %s", reader_name)
-    LOG.info("PRODUCT_NAME: %s", product_name)
-    LOG.info("NUM_PRODUCTS: %s", len(final_products))
-    LOG.info("NUM_DELETED_PRODUCTS: %s", len(removed_products))
+    LOG.interactive("READER_NAME: %s", reader_name)
```

Comment:
@evrose54 I think log.interactive statements will need to be added to "config_based.py" procflow also - which can be tested with one of the tests/scripts/*config* scripts.

Reply:
@mindyls Should that be a separate PR? Updating `config_based.py` I mean.

Reply:
Looks like this has been completed. Going to resolve this comment.